src/components/post/CommonPostCard.tsx

```diff
@@ -141,11 +187,11 @@ export default function PostCard(props: props) {
         <CardActions style={{ padding: "4px 8px" }}>
           <Box sx={{ flexGrow: 1 }}></Box>
 
-          {/* <Grow in={!hiddenPostDetail} style={{ transformOrigin: "0 0 0" }}>
-            <Button onClick={() => setOpenSnackBar(true)} variant="contained">
-              Join
+          <Grow in={!hiddenPostDetail} style={{ transformOrigin: "0 0 0" }}>
+            <Button onClick={() => joinPost()} variant="contained">
+              {!isUserJoin ? "Join" : "Cancel"}
             </Button>
-          </Grow> */}
+          </Grow>
 
           <ExpandMore expand={!hiddenPostDetail} onClick={handleExpandDetail}>
             <ArrowDownwardIcon />
```

Comment:
Not sure whether this code is what I am looking for or not. I think the arrow should be `black`
<img width="917" alt="image" src="https://user-images.githubusercontent.com/62459518/226544656-6ffc3dd3-08b9-4465-b197-c09123776666.png">

Reply:
Actually, there is no redesign on the index page in figma; however, ive just solved it.